src/modal/modal-stack.ts

```diff
@@ -22,6 +22,7 @@ import {NgbActiveModal, NgbModalRef} from './modal-ref';
 export class NgbModalStack {
   private _windowAttributes = ['ariaLabelledBy', 'backdrop', 'centered', 'keyboard', 'size', 'windowClass'];
   private _backdropAttributes = ['backdropClass'];
+  public modalRefs: NgbModalRef[] = [];
```

Comment:
Why is this public? Is it for tests only? If so please check my comment in the test file.